ninova_downloader.py

```diff
@@ -1,12 +1,13 @@
-import requests
-import bs4
+import datetime
 import getpass
 import os
-import datetime
-import shutil
+
+import bs4
+import requests
 
 '''Base URL'''
 url = 'https://ninova.itu.edu.tr'
+overwrite = 'n'
```

Comment:
Wouldn't it be better to use boolean values for this?
```suggestion
overwrite = False
```

Reply:
Sure why not